src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -258,6 +254,26 @@ public bool VerifyImport(TrackedDownload trackedDownload, List<ImportResult> imp
             return false;
         }
 
+        private void SendManualInteractionRequiredNotification(TrackedDownload trackedDownload)
+        {
+            if (!trackedDownload.HasNotifiedManualInteractionRequired)
+            {
+                var grabbedHistories = _historyService.FindByDownloadId(trackedDownload.DownloadItem.DownloadId).Where(h => h.EventType == EpisodeHistoryEventType.Grabbed).ToList();
+
+                if (grabbedHistories.Count == 0)
+                {
+                    grabbedHistories = new List<EpisodeHistory> { new EpisodeHistory() };
+                }
+
+                trackedDownload.HasNotifiedManualInteractionRequired = true;
+
+                var releaseInfo = new GrabbedReleaseInfo(grabbedHistories);
```

Comment:
Since I didn't like `grabbedHistories = new List<EpisodeHistory> { new EpisodeHistory() };` I did some digging and everything that handles `releaseInfo` later will ignore it when it's null so we can simply this to something like:

```suggestion
                trackedDownload.HasNotifiedManualInteractionRequired = true;

                var releaseInfo = grabbedHistories.Count > 0 ? new GrabbedReleaseInfo(grabbedHistories) : null;
```